CONTRIBUTING.md

```diff
@@ -1,20 +1,23 @@
 # Contributor's Guide
 
-We welcome pull requests from freeCodeCamp campers (our students) and seasoned JavaScript developers alike! Follow these steps to contribute:
+We welcome pull requests from members of the freeCodeCamp community and from veteran developers alike! Please follow these steps to contribute:
 
-1. Find an issue that needs assistance by searching for the [Help Wanted](https://github.com/freeCodeCamp/freeCodeCamp/labels/help%20wanted) tag.
+## Steps
 
-2. Let us know you are working on it by posting a comment on the issue.
+1. 🍴 [Fork this repo](https://github.com/freeCodeCamp/freeCodeCamp#fork-destination-box)
+2. 👀️ Follow the contributing guidelines outlined below.
+3. 🔧 Make some awesome changes!
+4. 👉 [Make a pull request](https://github.com/freeCodeCamp/freeCodeCamp/compare)
+5. 🎉 Get your pull request approved - success!
 
-3. Follow the instructions in this guide to start working on the issue.
+Or, If you've found a issue that is not on the board, [follow these steps](README.md#found-a-bug).
 
-Remember to feel free to ask for help in our [Contributors](https://gitter.im/FreeCodeCamp/Contributors) Gitter room.
+Feel free to ask for help in our public [Contributors chat room](https://gitter.im/FreeCodeCamp/Contributors). We are excited to help you contribute to any of the projects that you would like to work on.
```

Comment:
Feel free to ask for help in our public [Contributors chat room](https://gitter.im/FreeCodeCamp/Contributors), or on our [forum](https://forum.freecodecamp.org/c/contributors). We are excited to help you contribute to any of the projects that you would like to work on.

Reply:
Ah.. right we should add the form at all places. Thanks I'll update.